bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -735,24 +732,30 @@ std::string BCStateTran::convertMillisecToReadableStr(uint64_t ms) const {
 
   std::string str;
   if (days) {
-    str += std::to_string(days) + " day + ";
+    str += std::to_string(days) + ":";
+    legend += "D:";
```

Comment:
I think this function can be moved to #include "TimeUtils.hpp".
BStateTran.hpp already have above file included & using many time* related functions from it.

Reply:
I think I'll put TODO above it, to do it as part of refactoring.
This is not in grade for "TimeUtils.hpp" in my opinion.
The idea was to have a nice printouts in ST reporting.
For time utils (more general use case) , need to make it templated and be able to convert all unites to all unites, not only ms.